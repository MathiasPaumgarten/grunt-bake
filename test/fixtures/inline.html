<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/third.html title="shoes" tag="span" class="shoe-box")-->
	</body>
</html>