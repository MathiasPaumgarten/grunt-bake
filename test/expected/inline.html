<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<span class="shoe-box">shoes</span>
	</body>
</html>